app/src/main/java/com/duckduckgo/app/browser/omnibar/experiments/FadeOmnibarLayout.kt

```diff
@@ -308,8 +313,8 @@ class FadeOmnibarLayout @JvmOverloads constructor(
     }
 
     private fun onFindInPageShown() {
-        omniBarContentContainer.gone()
-        customTabToolbarContainerWrapper.gone()
+        omniBarContentContainer.hide()
+        customTabToolbarContainerWrapper.hide()
```

Comment:
This change ensures that when:
1. You open find in page.
2. Rotate the device.
3. Close find in page.

That the address bar is already in the correct position. Otherwise, if the view was `gone` while we're in find in page, it would need another render loop to re-adjust.